part1/chapter6/quiz1/questions/question_1.rb

```diff
@@ -27,3 +27,20 @@
 # * Note: You can assume that the user will always enter either an
 #   integer or `stop`.
 
+def sum_num
+total_numbers = []
```

Comment:
Nice!

It's convention to indent the lines of code after writing a method. 

The same goes for code following a while loop.